libasn1fix/asn1fix_cws.c

```diff
@@ -307,6 +307,8 @@ _asn1f_assign_cell_value(arg_t *arg, struct asn1p_ioc_row_s *row, struct asn1p_i
 			FATAL("Cannot find %s referenced by %s at line %d",
 				p, arg->expr->Identifier,
 				arg->expr->_lineno);
+			asn1p_ref_free(ref);
```

Comment:
Remove the following two lines : 
``
asn1p_ref_free(ref);
free(p);
```

I am not that acquainted with how to resolve this conflict in this scenario. 

I try to pull your newest master and rebase `fix_most_memory_leak' branch to it. But since these two lines are 'added' so no conflict indicated. I removed these two lines then push, but it does not trigger CI re-build in ordinary case.

I also try to merge 'fix_most_memory_leak' back to newly pull master branch, it indicates conflict, but I am not on the pull request branch. 

How should I do ?